api/config/config.exs

```
@@ -23,10 +23,15 @@ config :logger, :console,
 # Use Jason for JSON parsing in Phoenix
 config :phoenix, :json_library, Jason
 
-# Configures Redis
-redis_url = System.get_env("REDIS_URL") || "redis://localhost:6379"
+secrets_adapter = System.get_env("SECRETS_ADAPTER") || "REDIS"
```

We should throw an error on unsupported values maybe